addons/compat_rf/CfgVehicles.hpp

```diff
@@ -0,0 +1,26 @@
+class CfgVehicles {
+    // applies the offset to all RF Offroads which can have the optional Tank in the back
+    class Offroad_01_unarmed_base_F;
+    class Pickup_01_base_rf : Offroad_01_unarmed_base_F {
+        EXGVAR(field_rations,offset)[] = {-0.04, -2.45, -0.9};
+    };
+    
+    // Enable Water Source by Default
+    class C_IDAP_Pickup_rf;
+    class C_IDAP_Pickup_water_rf: C_IDAP_Pickup_rf {
+        EXGVAR(field_rations,waterSupply) = 500;
+    };
+    
+    class O_Truck_03_fuel_F;
+    class C_Truck_03_water_rf: O_Truck_03_fuel_F {
+        EXGVAR(field_rations,waterSupply) = 10000;
+        EXGVAR(field_rations,offset)[] = {0, -5.05, -0.3}; // Rear
+        //EXGVAR(field_rations,offset)[] = {1.25,-1.59,-0.58}; // Right Side Latch
```

Comment:
```suggestion
        EXGVAR(field_rations,offset)[] = {0, -5.05, -0.3};
```

The right side latch is more than likely the fuel tank.